### ISSUE TITLE
2025-04-29, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#154]
# Today's Arxiv Papers

(1) [AutoJudge: Judge Decoding Without Manual Annotation](http://arxiv.org/abs/2504.20039v1), 2025-04-28

(2) [SpatialReasoner: Towards Explicit and Generalizable 3D Spatial Reasoning](http://arxiv.org/abs/2504.20024v1), 2025-04-28

(3) [Better To Ask in English? Evaluating Factual Accuracy of Multilingual LLMs in English and Low-Resource Languages](http://arxiv.org/abs/2504.20022v1), 2025-04-28

(4) [Modular Machine Learning: An Indispensable Path towards New-Generation Large Language Models](http://arxiv.org/abs/2504.20020v1), 2025-04-28

(5) [Applying LLM-Powered Virtual Humans to Child Interviews in Child-Centered Design](http://dx.doi.org/10.1145/3713043.3731551), 2025-04-28

(6) [LLM-Generated Fake News Induces Truth Decay in News Ecosystem: A Case Study on Neural News Recommendation](http://dx.doi.org/10.1145/3726302.3730027), 2025-04-28

(7) [Towards Automated Scoping of AI for Social Good Projects](http://arxiv.org/abs/2504.20010v1), 2025-04-28

(8) [Chatbot Arena Meets Nuggets: Towards Explanations and Diagnostics in the Evaluation of LLM Responses](http://arxiv.org/abs/2504.20006v1), 2025-04-28

(9) [Knowledge Distillation of Domain-adapted LLMs for Question-Answering in Telecom](http://arxiv.org/abs/2504.20000v1), 2025-04-28

(10) [TD-EVAL: Revisiting Task-Oriented Dialogue Evaluation by Combining Turn-Level Precision with Dialogue-Level Comparisons](http://arxiv.org/abs/2504.19982v1), 2025-04-28

(11) [Accurate and Diverse LLM Mathematical Reasoning via Automated PRM-Guided GFlowNets](http://arxiv.org/abs/2504.19981v1), 2025-04-28

(12) [From Concept to Practice: an Automated LLM-aided UVM Machine for RTL Verification](http://arxiv.org/abs/2504.19959v1), 2025-04-28

(13) [Securing Agentic AI: A Comprehensive Threat Model and Mitigation Framework for Generative AI Agents](http://arxiv.org/abs/2504.19956v1), 2025-04-28

(14) [Enhancing Surgical Documentation through Multimodal Visual-Temporal Transformers and Generative AI](http://arxiv.org/abs/2504.19918v1), 2025-04-28

(15) [Can AI Agents Design and Implement Drug Discovery Pipelines?](http://arxiv.org/abs/2504.19912v1), 2025-04-28

(16) [GenCLS++: Pushing the Boundaries of Generative Classification in LLMs Through Comprehensive SFT and RL Studies Across Diverse Datasets](http://arxiv.org/abs/2504.19898v1), 2025-04-28

(17) [CineVerse: Consistent Keyframe Synthesis for Cinematic Scene Composition](http://arxiv.org/abs/2504.19894v1), 2025-04-28

(18) [semi-PD: Towards Efficient LLM Serving via Phase-Wise Disaggregated Computation and Unified Storage](http://arxiv.org/abs/2504.19867v1), 2025-04-28

(19) [CoherenDream: Boosting Holistic Text Coherence in 3D Generation via Multimodal Large Language Models Feedback](http://arxiv.org/abs/2504.19860v1), 2025-04-28

(20) [The Automation Advantage in AI Red Teaming](http://arxiv.org/abs/2504.19855v1), 2025-04-28

(21) [LLM-Powered GUI Agents in Phone Automation: Surveying Progress and Prospects](http://arxiv.org/abs/2504.19838v1), 2025-04-28

(22) [PhenoAssistant: A Conversational Multi-Agent AI System for Automated Plant Phenotyping](http://arxiv.org/abs/2504.19818v1), 2025-04-28

(23) [Can a Crow Hatch a Falcon? Lineage Matters in Predicting Large Language Model Performance](http://arxiv.org/abs/2504.19811v1), 2025-04-28

(24) [Prompt Injection Attack to Tool Selection in LLM Agents](http://arxiv.org/abs/2504.19793v1), 2025-04-28

(25) [Moral Reasoning Across Languages: The Critical Role of Low-Resource Languages in LLMs](http://arxiv.org/abs/2504.19759v1), 2025-04-28

(26) [Reconstructing Context: Evaluating Advanced Chunking Strategies for Retrieval-Augmented Generation](http://arxiv.org/abs/2504.19754v1), 2025-04-28

(27) [FineQ: Software-Hardware Co-Design for Low-Bit Fine-Grained Mixed-Precision Quantization of LLMs](http://arxiv.org/abs/2504.19746v1), 2025-04-28

(28) [LLM-Assisted Automated Deductive Coding of Dialogue Data: Leveraging Dialogue-Specific Characteristics to Enhance Contextual Understanding](http://arxiv.org/abs/2504.19734v1), 2025-04-28

(29) [Evaluate-and-Purify: Fortifying Code Language Models Against Adversarial Attacks Using LLM-as-a-Judge](http://arxiv.org/abs/2504.19730v1), 2025-04-28

(30) [Taming the Titans: A Survey of Efficient LLM Inference Serving](http://arxiv.org/abs/2504.19720v1), 2025-04-28

(31) [Guided Tensor Lifting](http://arxiv.org/abs/2504.19705v1), 2025-04-28

(32) [From LLM Reasoning to Autonomous AI Agents: A Comprehensive Review](http://arxiv.org/abs/2504.19678v1), 2025-04-28

(33) [Annif at SemEval-2025 Task 5: Traditional XMTC augmented by LLMs](http://arxiv.org/abs/2504.19675v1), 2025-04-28

(34) [$\texttt{SAGE}$: A Generic Framework for LLM Safety Evaluation](http://arxiv.org/abs/2504.19674v1), 2025-04-28

(35) [A Tripartite Perspective on GraphRAG](http://arxiv.org/abs/2504.19667v1), 2025-04-28

(36) [Decentralization of Generative AI via Mixture of Experts for Wireless Networks: A Comprehensive Survey](http://arxiv.org/abs/2504.19660v1), 2025-04-28

(37) [Intelligent4DSE: Optimizing High-Level Synthesis Design Space Exploration with Graph Neural Networks and Large Language Models](http://arxiv.org/abs/2504.19649v1), 2025-04-28

(38) [Fitness Landscape of Large Language Model-Assisted Automated Algorithm Search](http://arxiv.org/abs/2504.19636v1), 2025-04-28

(39) [Scene2Hap: Combining LLMs and Physical Modeling for Automatically Generating Vibrotactile Signals for Full VR Scenes](http://arxiv.org/abs/2504.19611v1), 2025-04-28

(40) [Coreference Resolution for Vietnamese Narrative Texts](http://arxiv.org/abs/2504.19606v1), 2025-04-28

(41) [GVPO: Group Variance Policy Optimization for Large Language Model Post-Training](http://arxiv.org/abs/2504.19599v1), 2025-04-28

(42) [Mapping the Italian Telegram Ecosystem](http://arxiv.org/abs/2504.19594v1), 2025-04-28

(43) [Semantic Packet Aggregation for Token Communication via Genetic Beam Search](http://arxiv.org/abs/2504.19591v1), 2025-04-28

(44) [Graph-Based Spectral Decomposition for Parameter Coordination in Language Model Fine-Tuning](http://arxiv.org/abs/2504.19583v1), 2025-04-28

(45) [m-KAILIN: Knowledge-Driven Agentic Scientific Corpus Distillation Framework for Biomedical Large Language Models Training](http://arxiv.org/abs/2504.19565v1), 2025-04-28

(46) [Quantifying Memory Utilization with Effective State-Size](http://arxiv.org/abs/2504.19561v1), 2025-04-28

(47) [Detecting Effects of AI-Mediated Communication on Language Complexity and Sentiment](http://dx.doi.org/10.1145/3701716.3717543), 2025-04-28

(48) [DEEMO: De-identity Multimodal Emotion Recognition and Reasoning](http://arxiv.org/abs/2504.19549v1), 2025-04-28

(49) [LR-IAD:Mask-Free Industrial Anomaly Detection with Logical Reasoning](http://arxiv.org/abs/2504.19524v1), 2025-04-28

(50) [Security Steerability is All You Need](http://arxiv.org/abs/2504.19521v1), 2025-04-28

(51) [Bullet: Boosting GPU Utilization for LLM Serving via Dynamic Spatial-Temporal Orchestration](http://arxiv.org/abs/2504.19516v1), 2025-04-28

(52) [Evolution of Cooperation in LLM-Agent Societies: A Preliminary Study Using Different Punishment Strategies](http://arxiv.org/abs/2504.19487v1), 2025-04-28

(53) [Improving Reasoning Performance in Large Language Models via Representation Engineering](http://arxiv.org/abs/2504.19483v1), 2025-04-28

(54) [An Automated Reinforcement Learning Reward Design Framework with Large Language Model for Cooperative Platoon Coordination](http://arxiv.org/abs/2504.19480v1), 2025-04-28

(55) [BRIDGE: Benchmarking Large Language Models for Understanding Real-world Clinical Practice Text](http://arxiv.org/abs/2504.19467v1), 2025-04-28

(56) [Do Automatic Comment Generation Techniques Fall Short? Exploring the Influence of Method Dependencies on Code Understanding](http://arxiv.org/abs/2504.19459v1), 2025-04-28

(57) [Towards Long Context Hallucination Detection](http://arxiv.org/abs/2504.19457v1), 2025-04-28

(58) [Masked Language Prompting for Generative Data Augmentation in Few-shot Fashion Style Recognition](http://arxiv.org/abs/2504.19455v1), 2025-04-28

(59) [Artificial Intelligence in Number Theory: LLMs for Algorithm Generation and Neural Networks for Conjecture Verification](http://arxiv.org/abs/2504.19451v1), 2025-04-28

(60) [R-Sparse: Rank-Aware Activation Sparsity for Efficient LLM Inference](http://arxiv.org/abs/2504.19449v1), 2025-04-28

(61) [Systematic Bias in Large Language Models: Discrepant Response Patterns in Binary vs. Continuous Judgment Tasks](http://arxiv.org/abs/2504.19445v1), 2025-04-28

(62) [Large Language Models are Qualified Benchmark Builders: Rebuilding Pre-Training Datasets for Advancing Code Intelligence Tasks](http://arxiv.org/abs/2504.19444v1), 2025-04-28

(63) [Context-Guided Dynamic Retrieval for Improving Generation Quality in RAG Models](http://arxiv.org/abs/2504.19436v1), 2025-04-28

(64) [MER 2025: When Affective Computing Meets Large Language Models](http://arxiv.org/abs/2504.19423v1), 2025-04-28

(65) [Mem0: Building Production-Ready AI Agents with Scalable Long-Term Memory](http://arxiv.org/abs/2504.19413v1), 2025-04-28

(66) [Context Selection and Rewriting for Video-based EducationalQuestion Generation](http://arxiv.org/abs/2504.19406v1), 2025-04-28

(67) [ICL CIPHERS: Quantifying "Learning'' in In-Context Learning via Substitution Ciphers](http://arxiv.org/abs/2504.19395v1), 2025-04-28

(68) [LLMs for Engineering: Teaching Models to Design High Powered Rockets](http://arxiv.org/abs/2504.19394v1), 2025-04-27

(69) [Bi-directional Model Cascading with Proxy Confidence](http://arxiv.org/abs/2504.19391v1), 2025-04-27

(70) [From Inductive to Deductive: LLMs-Based Qualitative Data Analysis in Requirements Engineering](http://arxiv.org/abs/2504.19384v1), 2025-04-27

(71) [Contextual Online Uncertainty-Aware Preference Learning for Human Feedback](http://arxiv.org/abs/2504.19342v1), 2025-04-27

(72) [OpenFOAMGPT 2.0: end-to-end, trustworthy automation for computational fluid dynamics](http://arxiv.org/abs/2504.19338v1), 2025-04-27

(73) [Unified Multi-Task Learning & Model Fusion for Efficient Language Model Guardrailing](http://arxiv.org/abs/2504.19333v1), 2025-04-27

(74) [BrowseComp-ZH: Benchmarking Web Browsing Ability of Large Language Models in Chinese](http://arxiv.org/abs/2504.19314v1), 2025-04-27

(75) [AndroidGen: Building an Android Language Agent under Data Scarcity](http://arxiv.org/abs/2504.19298v1), 2025-04-27

(76) [Small Models, Big Tasks: An Exploratory Empirical Study on Small Language Models for Function Calling](http://arxiv.org/abs/2504.19277v1), 2025-04-27

(77) [VIST-GPT: Ushering in the Era of Visual Storytelling with LLMs?](http://arxiv.org/abs/2504.19267v1), 2025-04-27

(78) [The Convergent Ethics of AI? Analyzing Moral Foundation Priorities in Large Language Models with a Multi-Framework Approach](http://arxiv.org/abs/2504.19255v1), 2025-04-27

(79) [Uncertainty Quantification for Language Models: A Suite of Black-Box, White-Box, LLM Judge, and Ensemble Scorers](http://arxiv.org/abs/2504.19254v1), 2025-04-27

(80) [Representing and querying data tensors in RDF and SPARQL](http://arxiv.org/abs/2504.19224v1), 2025-04-27

(81) [AlphaFuse: Learn ID Embeddings for Sequential Recommendation in Null Space of Language Embeddings](http://dx.doi.org/10.1145/3726302.3729894), 2025-04-27

(82) [WuNeng: Hybrid State with Attention](http://arxiv.org/abs/2504.19191v1), 2025-04-27

(83) [Hierarchical Attention Generates Better Proofs](http://arxiv.org/abs/2504.19188v1), 2025-04-27

(84) [Segmenting Objectiveness and Task-awareness Unknown Region for Autonomous Driving](http://arxiv.org/abs/2504.19183v1), 2025-04-27

(85) [SPC: Evolving Self-Play Critic via Adversarial Games for LLM Reasoning](http://arxiv.org/abs/2504.19162v1), 2025-04-27

(86) [Muyan-TTS: A Trainable Text-to-Speech Model Optimized for Podcast Scenarios with a $50K Budget](http://arxiv.org/abs/2504.19146v1), 2025-04-27

(87) [ChiseLLM: Unleashing the Power of Reasoning LLMs for Chisel Agile Hardware Development](http://arxiv.org/abs/2504.19144v1), 2025-04-27

(88) [APE-Bench I: Towards File-level Automated Proof Engineering of Formal Math Libraries](http://arxiv.org/abs/2504.19110v1), 2025-04-27

(89) [A Multi-Language Perspective on the Robustness of LLM Code Generation](http://arxiv.org/abs/2504.19108v1), 2025-04-27

(90) [Privacy-Preserving Federated Embedding Learning for Localized Retrieval-Augmented Generation](http://arxiv.org/abs/2504.19101v1), 2025-04-27

(91) [VeriDebug: A Unified LLM for Verilog Debugging via Contrastive Embedding and Guided Correction](http://arxiv.org/abs/2504.19099v1), 2025-04-27

(92) [Efficient Reasoning for LLMs through Speculative Chain-of-Thought](http://arxiv.org/abs/2504.19095v1), 2025-04-27

(93) [CipherBank: Exploring the Boundary of LLM Reasoning Capabilities through Cryptography Challenges](http://arxiv.org/abs/2504.19093v1), 2025-04-27

(94) [LLM-Evaluation Tropes: Perspectives on the Validity of LLM-Evaluations](http://arxiv.org/abs/2504.19076v1), 2025-04-27

(95) [HoloDx: Knowledge- and Data-Driven Multimodal Diagnosis of Alzheimer's Disease](http://arxiv.org/abs/2504.19075v1), 2025-04-27

(96) [ClimaEmpact: Domain-Aligned Small Language Models and Datasets for Extreme Weather Analytics](http://arxiv.org/abs/2504.19066v1), 2025-04-27

(97) [Hallucinations and Key Information Extraction in Medical Texts: A Comprehensive Assessment of Open-Source Large Language Models](http://arxiv.org/abs/2504.19061v1), 2025-04-27

(98) [Generative AI for Character Animation: A Comprehensive Survey of Techniques, Applications, and Future Directions](http://arxiv.org/abs/2504.19056v1), 2025-04-27

(99) [Calibrating Translation Decoding with Quality Estimation on LLMs](http://arxiv.org/abs/2504.19044v1), 2025-04-26

(100) ["I Would Have Written My Code Differently'': Beginners Struggle to Understand LLM-Generated Code](http://dx.doi.org/10.1145/3696630.3731663), 2025-04-26
